main/rhash/APKBUILD

```diff
@@ -1,34 +1,40 @@
 # Maintainer: Przemyslaw Pawelczyk <przemoc@zoho.com>
 pkgname=rhash
-pkgver=1.3.5
-pkgrel=1
+pkgver=1.3.6
+pkgrel=0
 pkgdesc="Utility for calculation and verification of hash sums and magnet links"
-url="http://rhash.anz.ru/"
+url="http://rhash.anz.ru"
 arch="all"
 license="custom"
+makedepends="libressl-dev"
 subpackages="$pkgname-doc $pkgname-dev $pkgname-libs"
-source="$pkgname-$pkgver.tar.gz::http://downloads.sourceforge.net/$pkgname/$pkgname-$pkgver-src.tar.gz"
+source="http://downloads.sourceforge.net/$pkgname/$pkgname-$pkgver-src.tar.gz"
 builddir="$srcdir/RHash-$pkgver"
 
+prepare() {
+	# include endian.h on musl
```

Comment:
Please always call `default_prepare` in custom `prepare()` function.

Reply:
@jirutka Noted and fixed.
Thanks.